public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:400,500,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:100,300,400,500,700" rel="stylesheet">
```

Comment:
are we using all 5 of these weights? i think google says that can slow down load times a little bit, so might not be possible but it would be ideal if we can slim that down a bit